cwl/src/test/resources/bad2.cwl

```diff
@@ -0,0 +1 @@
+gibberish
```

Comment:
They are the same.  The test was originally intended to make sure it picked up 2 separate errors, as two errors re: `bad.cwl` could feasibly be the same error twice. 

I see now that I'm not asserting the 2 separate error condition.  I'll fix it up.